webpack/assets/javascripts/react_app/routes/RegistrationCommands/RegistrationCommandsPage/components/fields/DownloadUtility.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import {
+  FormGroup,
+  FormSelect,
+  FormSelectOption,
+} from '@patternfly/react-core';
+
+import { translate as __ } from '../../../../../common/I18n';
+
+export const DownloadUtilities = {
+  curl: 'curl',
+  wget: 'wget',
+};
+const DownloadUtility = ({
+  downloadUtility,
+  handleDownloadUtility,
+  isLoading,
+}) => (
+  <FormGroup label={__('Download utility')} fieldId="reg_download_utility">
+    <FormSelect
+      ouiaId="reg_download_utility"
+      value={downloadUtility}
+      onChange={v => handleDownloadUtility(v)}
+      className="without_select2"
+      id="reg_download_utility"
+      isDisabled={isLoading}
+    >
+      {_.map(DownloadUtilities, (key, value) => (
```

Comment:
I'm not much JS proficient, what does the `_.map` do?

Reply:
The `_` refers to the Lodash (and Underscore) JS libraries, which provide functions for manipulating arrays, lists, etc. I initially thought it is used in Foreman. But I saw it is only pulled in as dependency.

I now use the default `map()` function. I also switched to using a list for the `DownloadUtilities` as there is no need for using a dictionary.